docs/contributing/commit-discipline.md

```diff
@@ -279,6 +279,23 @@ might worry about it breaking.
   you developed this commit or pull request, like "First I tried X" or
   "I changed Y".
 
+#### Mentioning other contributors
+
+You can
+[credit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)
+co-authors on a commit by adding a `Co-authored-by:` line after a blank line at
+the end of your commit message:
+
+    Co-authored-by: Greg Price <greg@zulip.com>
+
+You can also add other notes, such as `Reported-by:`, `Debugged-by:` or
+`Suggested-by:`, but we do not generally do so in this project.
+
+Never @-mention a contributor in a commit message, as GitHub will turn this into
```

Comment:
I think this would benefit from being marked with some kind of sub-heading of its own. Otherwise it's easy to miss as just more prose on the `Co-authored-by:` thing.

One version of that would be to put the head phrase in bold (like Karl suggested at https://github.com/zulip/zulip/pull/25768#issuecomment-1564940476 ):
```suggestion
**Never @-mention a contributor in a commit message**, as GitHub will turn this into
```

which is the style I've used for the smallest-granularity subheadings in the [zulip-mobile style guide](https://github.com/zulip/zulip-mobile/blob/main/docs/style.md).

On this page as it exists, there are several different styles being used, one of which is bold, e.g. for "complete sentence". (I think the page could also use more headings and such explicit structure in general, but that can be left out of scope.)